ui/package.json

```diff
@@ -44,6 +44,7 @@
     "@babel/plugin-proposal-object-rest-spread": "^7.12.1",
     "@babel/plugin-transform-block-scoping": "^7.12.1",
     "@ember/optional-features": "^2.0.0",
+    "@ember/render-modifiers": "^1.0.2",
```

Comment:
honestly, I'm surprised we haven't needed this earlier.  Thanks for bringing it into the dependencies.